src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs

```diff
@@ -210,6 +210,12 @@ public class DurableTaskOptions
         /// <value>Assembly qualified class name that implements <see cref="ILifeCycleNotificationHelper">ILifeCycleNotificationHelper</see>.</value>
         public string CustomLifeCycleNotificationHelperType { get; set; }
 
+        /// <summary>
+        /// Gets or sets the maximum queue polling interval.
+        /// </summary>
+        /// <value>Maximum interval for polling control and work-item queues.</value>
+        public TimeSpan MaxQueuePollingInterval { get; set; } = TimeSpan.FromSeconds(30);
```

Comment:
Let's also add this property value to the `GetDebugString()` method further below. This is important for diagnostics,

Reply:
Done!